<?xml version="1.0"?>

<launch>

  <!-- Adapters parameters -->
    <!-- Define the paths of the csv files to read -->
    <param name="back_csv_path" value="/data/volunteer_01/IMUs/back.csv"/>
    <param name="lla_csv_path" value="/data/volunteer_01/IMUs/lla.csv"/>
    <param name="lua_csv_path" value="/data/volunteer_01/IMUs/lua.csv"/>
    <param name="rla_csv_path" value="/data/volunteer_01/IMUs/rla.csv"/>
    <param name="rt_csv_path" value="/data/volunteer_01/IMUs/rt.csv"/>
    <param name="rua_csv_path" value="/data/volunteer_01/IMUs/rua.csv"/>


  <!-- Final classificator parameters -->
    <!-- Define the tolerance (in ms) allowed when testing the coherence of the timestamps -->
    <param name="tolerance_in_ms" value="100"/>

    <!-- Define if you want to write the system's output on a csv file -->
    <param name="write_on_csv" value="True" type="bool"/>

    <!-- Define the csv file path -->
    <param name="output_csv_path" value="output/classification.csv"/>


  <!-- Include other launch files -->
  <include file="$(find group_project)/launch/adapters.launch"></include>
  <include file="$(find group_project)/launch/final_classificator.launch"></include>

</launch>